core/src/messages.cpp

```diff
@@ -342,20 +342,19 @@ namespace {
  * used for lookup directly. Past this point, oneseismic only works in the
  * cartesian grid and no longer cares about line numbers.
  */
-void to_cartesian_inplace(
-    const std::vector< int >& labels,
-    std::vector< int >& xs)
-noexcept (false) {
+void to_cartesian_inplace(const std::vector<int>& labels, std::vector<int>& xs,
```

Comment:
Passing a parameter only for the error message feels a bit "off" to me.

Reply:
I am struggling to find a good design to address that, so will continue to struggle next month :smile: